tests/Doctrine/DBAL/Migrations/Tests/Functional/FunctionalTest.php

```diff
@@ -13,6 +16,7 @@
 use Doctrine\DBAL\Migrations\Tests\Stub\Functional\MigrationModifySchemaInPreAndPost;
 use Doctrine\DBAL\Schema\Schema;
 use Doctrine\DBAL\Migrations\Configuration\Configuration;
+use \Mockery as m;
```

Comment:
It's at least already used in the SqlFileWriter https://github.com/doctrine/migrations/blob/master/tests/Doctrine/DBAL/Migrations/Tests/SqlFileWriterTest.php#L24